src/Stack/Options/BenchParser.hs

```diff
@@ -19,7 +19,7 @@ benchOptsParser hide0 = BenchmarkOptsMonoid
                                  help ("Forward BENCH_ARGS to the benchmark suite. " <>
                                        "Supports templates from `cabal bench`") <>
                                  hide))
-        <*> optionalFirst (switch (long "no-run-benchmarks" <>
+        <*> optionalFirst (flag' True (long "no-run-benchmarks" <>
```

Comment:
Can you add a comment to the changelog about the bugfix?

Reply:
done